src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -306,8 +309,11 @@ public async Task<ListResponseModel<CipherMiniDetailsResponseModel>> GetOrganiza
         {
             throw new NotFoundException();
         }
-
-        var allOrganizationCiphers = await _organizationCiphersQuery.GetAllOrganizationCiphers(organizationId);
+        var allOrganizationCiphers = _featureService.IsEnabled(FeatureFlagKeys.CreateDefaultLocation)
```

Comment:
💭 I know this is the only flag we have at the moment, but the name doesn't seem to properly reflect the behavior being modified here. We might want to create our own Vault flag related to these item organization changes.